packages/frontend/core/src/desktop/dialogs/setting/setting-sidebar/index.tsx

```diff
@@ -280,7 +280,7 @@ const WorkspaceListItem = ({
     onClick('workspace:preference');
   }, [onClick]);
 
-  const showBilling = information?.isTeam && information?.isOwner;
+  const showBilling = information?.isOwner;
```

Comment:
team only

Reply:
Oh, I changed it when debugging just now, and forgot to change it back